src/ProjectOrigin.WalletSystem.IntegrationTests/Repositories/WalletRepositoryTests.cs

```diff
@@ -0,0 +1,106 @@
+using Dapper;
+using FluentAssertions;
+using ProjectOrigin.WalletSystem.Server.Database;
+using ProjectOrigin.WalletSystem.Server.Database.Mapping;
+using ProjectOrigin.WalletSystem.Server.HDWallet;
+using ProjectOrigin.WalletSystem.Server.Models;
+using ProjectOrigin.WalletSystem.Server.Repositories;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ProjectOrigin.WalletSystem.IntegrationTests.Repositories;
+
+public class WalletRepositoryTests : IClassFixture<PostgresDatabaseFixture>
+{
+    private readonly PostgresDatabaseFixture _dbFixture;
+    private readonly Secp256k1Algorithm _algorithm;
+
+    public WalletRepositoryTests(PostgresDatabaseFixture dbFixture)
+    {
+        _algorithm = new Secp256k1Algorithm();
+        _dbFixture = dbFixture;
+
+        SqlMapper.AddTypeHandler(new HDPrivateKeyTypeHandler(_algorithm));
+        SqlMapper.AddTypeHandler(new HDPublicKeyTypeHandler(_algorithm));
```

Comment:
The same two lines is in UnitOfWorkTests. I think it is easy to forget these lines of code, when writing more tests.